minidec/main.rs

```diff
@@ -101,8 +101,7 @@ fn main() {
             let mut cse = CSE::new(&mut rfn.ssa);
             cse.run();
         }
-        {
-            // Value Set Analysis
+        if false {
```

Comment:
yep, this VSA only concentrates on x86/x64 and cause some panics, which I don't why. So I disabled it temporarily.